components/shoplist-card/index.js

```diff
@@ -18,31 +18,67 @@ export const ShopCard = (props) => {
         <img src={path.join('./assets', product.image)} alt={product.name} />
       </Link>
       <div className="shop-card__content">
-        <p> {product.name}</p>
-        <p> {product.price}</p>
-        <button onClick={() => handleAddProduct(product.name)}> + </button>
-        <button onClick={() => handleRemoveProduct(product.name)}> - </button>
-        <p> Quantity : {props.quantity}</p>
+        <p className="shop-card-product-name"> {product.name}</p>
+        <p className="shop-card-product-price"> {product.price}</p>
+        <div className="shop-card-product-qnty">
+          <button onClick={() => handleAddProduct(product.name)}> + </button>
+          <span>{props.quantity}</span>
+          <button onClick={() => handleRemoveProduct(product.name)}> - </button>
+        </div>
       </div>
       <style jsx>{`
         .shop-card {
-          display: inline-block;
+          display: flex;
+          flex-direction: column;
+          align-items: center;
+          justify-content: space-between;
           margin: 10px;
           border: 1px solid #eee;
           box-shadow: 0 2px 2px #ccc;
-          width: 400px;
           text-align: center;
+          min-width: 300px;
+          border-radius: 30px;
+          transition: 0.2s;
+        }
+        .shop-card:hover {
+          box-shadow: 0px 4px 10px #ccc;
+          cursor: pointer;
         }
         .shop-card img {
-          width: 100px;
-          height: 100px;
+          width: 80%;
+          flex: 1;
         }
         .shop-card__content {
-          padding: 1rem;
           text-align: center;
+          width: 100%;
+          padding: 1em;
```

Comment:
It'll be great if you can go through this blog and make required chages for your task.
https://css-tricks.com/rems-ems/

In case there are any upper level refactoring required which is not related to your task, skip it. We can send a separate PR for those changes.